mlir/test/Target/LLVMIR/omptarget-fortran-allocatable-record-type-mapping-host.mlir

```diff
@@ -0,0 +1,66 @@
+// RUN: mlir-translate -mlir-to-llvmir %s | FileCheck %s
+
+// This test checks the offload sizes, map types and base pointers and pointers
+// provided to the OpenMP kernel argument structure are correct when lowering
+// to LLVM-IR from MLIR when performing explicit member mapping of a record type
+// that includes fortran allocatables in various locations of the record types
+// hierarchy.
```

Comment:
Nit: Could this test be described in terms of omp, llvm or builtin dialect MLIR operations and types? This is a test of the OpenMP dialect, so it's odd to find references to Fortran allocatables when the important thing here is that we're translating `omp.map.info` operations taking arguments of some specific types or perhaps containing a `var_ptr_ptr` argument (not sure what exactly the new thing being tested by this is, in terms of MLIR input).

In general, it looks like names in the test could be simplified too, since we don't need them to look like the raw output of Flang lowering.